README.md

```diff
@@ -402,12 +402,34 @@ Here are some tips for fixing unexpected issues.
 
 - Distributed Keras tuner support.
 
-## Contributing
+# CloudTuner
```

Comment:
Nit: having these descriptions at the end of the README is not very readable or discoverable. I would recommend having a per-folder README file with the content below, and in the main README, the different projects would be listed in the intro (with links to their README). The remainder of the main README can be about the `run` API.

Reply:
Updated the README as per this.